pkg/apic/apiserver/models/api/v1/types.go

```diff
@@ -49,8 +49,12 @@ type Reference struct {
 	Kind string `json:"kind,omitempty"`
 	// The name of the resource that it is referring to.
 	Name string `json:"name,omitempty"`
+	// The kind of the scope of the resource that it is referring to.
+	ScopeKind string `json:"scopeKind,omitempty"`
 	// The name of the scope of the resource that it is referring to.
-	Scope string `json:"scope,omitempty"`
+	ScopeName string `json:"scopeName,omitempty"`
```

Comment:
I hope we are not using "Scope" field in agents, could result in compilation failures